ceph-releases/kraken/ubuntu/16.04/daemon/debug.sh

```diff
@@ -19,14 +19,29 @@ for option in $(comma_to_space ${DEBUG}); do
       # This is just a safeguard
       pushd / > /dev/null
       wget -q $(extract_param $option) -O patch.tar
+
+      # Let's try to find if the tarball have the / in a sub-directory
```

Comment:
s/Let's try to find if the tarball have the / in a sub-directory/Let's find out if the tarball has the / in a sub-directory/